connectors/src/connectors/slack/bot.ts

```diff
@@ -223,16 +223,13 @@ async function botAnswerMessage(
     connector
   );
 
-  const dustAPI = new DustAPI(
-    {
+  const dustAPI = new DustAPI({
+    credentials: {
       workspaceId: connector.workspaceId,
       apiKey: connector.workspaceAPIKey,
     },
     logger,
-    {
-      useLocalInDev: true,
-    }
-  );
+  });
```

Comment:
in `connectors/`, we want to rely on `DUST_FRONT_API` all the time. We are using the `DustAPI` client like an external developer would use it.